flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogMapState.java

```diff
@@ -83,6 +84,16 @@ public UV setValue(UV value) {
                 }
                 return oldValue;
             }
+
+            @Override
+            public boolean equals(Object o) {
```

Comment:
This fix should not be related to Junit5 migration, right?
It's better to seprate them into 2 commits.